### ISSUE TITLE
Can't deploy plugin on Github

### ISSUE THREAD

**Issue by sekerez** [MihajloNesic/jekyll-pdf-embed#8]
Hi! I love your plugin, and it works fine when I run it locally, but deployment from github pages fails when I install only "jekyll-pdf-embed" as specified in the "RubyGems" section of README.

The deployment error looks like this:

```
Artifact URL: https://pipelines.actions.githubusercontent.com/0DhVIEsaJKQ9Eckd3ESwERMVol5zbAbXwWA0Us3Mn0JCN8u25X/_apis/pipelines/workflows/1755437989/artifacts?api-version=6.0-preview
{"count":0,"value":[]}
Failed to create deployment for dd662585f7cca4bad6a8268a3a1b54181f0d3398.
Error: Error: No uploaded artifact was found! Please check if there are any errors at build step.
Error: Error: No uploaded artifact was found! Please check if there are any errors at build step.
```

So I tried to instead install the plugin as specified in the "Github RubyGems registry" section, but when I bundle it locally I get this error:

```
Authentication is required for rubygems.pkg.github.com.
Please supply credentials for this source. You can do this by running:
`bundle config set --global rubygems.pkg.github.com username:password`
or by storing the credentials in the `BUNDLE_RUBYGEMS__PKG__GITHUB__COM` environment variable
```

I tried using my own github credentials, but I was denied access. I also realized that the link "https://rubygems.pkg.github.com/mihajlonesic" yields a 404 error.

Am I doing something wrong? I really like your plugin, is there any way to use it on github?

**Comment by andrei-radulescu-banu** [comment]
I had a similar error on Github Pages when adding a pdf tag in a post:

```
Liquid Exception: Liquid syntax error (line 1): Unknown tag 'pdf' in /github/workspace/_posts/2022-02-26-traian-popovici.md
/usr/local/bundle/gems/liquid-4.0.3/lib/liquid/document.rb:23:in `unknown_tag': Liquid syntax error (line 1): Unknown tag 'pdf' (Liquid::SyntaxError)
	from /usr/local/bundle/gems/liquid-4.0.3/lib/liquid/document.rb:11:in `block in parse'
	from /usr/local/bundle/gems/liquid-4.0.3/lib/liquid/block_body.rb:32:in `parse'
	from /usr/local/bundle/gems/liquid-4.0.3/lib/liquid/document.rb:10:in `parse'
	from /usr/local/bundle/gems/liquid-4.0.3/lib/liquid/document.rb:5:in `parse'
	from /usr/local/bundle/gems/liquid-4.0.3/lib/liquid/template.rb:132:in `parse'
	from /usr/local/bundle/gems/liquid-4.0.3/lib/liquid/template.rb:116:in `parse'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/liquid_renderer/file.rb:13:in `block in parse'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/liquid_renderer/file.rb:49:in `measure_time'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/liquid_renderer/file.rb:12:in `parse'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:121:in `render_liquid'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:79:in `render_document'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:62:in `run'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/site.rb:479:in `render_regenerated'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/site.rb:464:in `block (2 levels) in render_docs'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/site.rb:463:in `each'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/site.rb:463:in `block in render_docs'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/site.rb:462:in `each_value'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/site.rb:462:in `render_docs'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/site.rb:191:in `render'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/site.rb:71:in `process'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/command.rb:28:in `process_site'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/commands/build.rb:65:in `build'
	from /usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/commands/build.rb:36:in `process'
	from /usr/local/bundle/gems/github-pages-223/bin/github-pages:70:in `block (3 levels) in <top (required)>'
	from /usr/local/bundle/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `block in execute'
	from /usr/local/bundle/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `each'
	from /usr/local/bundle/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `execute'
	from /usr/local/bundle/gems/mercenary-0.3.6/lib/mercenary/program.rb:42:in `go'
	from /usr/local/bundle/gems/mercenary-0.3.6/lib/mercenary.rb:19:in `program'
	from /usr/local/bundle/gems/github-pages-223/bin/github-pages:6:in `<top (required)>'
	from /usr/local/bundle/bin/github-pages:23:in `load'
	from /usr/local/bundle/bin/github-pages:23:in `<main>'
```

My setup: I added this to Gemfile
```
# From https://github.com/MihajloNesic/jekyll-pdf-embed
source "https://rubygems.pkg.github.com/mihajlonesic" do
  gem "jekyll-pdf-embed", "1.1.2.1"
end
```
And added this to _config.yml:
```
plugins:
 - jekyll-pdf-embed # From https://github.com/MihajloNesic/jekyll-pdf-embed
```

**Comment by andrei-radulescu-banu** [comment]
Thanks for writing the plugin :)

Still does not work. This is what I tried:
- I forked your  https://github.com/MihajloNesic/minimal-mistakes-pdf-example as https://github.com/Bitdribble/minimal-mistakes-pdf-example
- In repo Settings->Pages, configured GitHub Pages site to build it from the master branch 
- Pages failed to build with this error:

```
/usr/local/bundle/gems/jekyll-3.9.0/lib/jekyll/theme.rb:84:in `rescue in gemspec': The minimal-mistakes-jekyll theme could not be found. (Jekyll::Errors::MissingDependencyException)
```

- To fix it I changed the theme from minimal-mistakes-jekyll to Minimal. That got past the error above. The new error is:

```
Liquid Exception: Liquid syntax error (line 7): Unknown tag 'pdf' in /github/workspace/_posts/2021-10-23-jekyll-pdf-embed-test.markdown
```

Which is what I was seeing on my site. It's unclear to me how your site works with Pages, and my fork does not. I thought the fork should 'just work'.
